openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -147,6 +148,12 @@ class AccountViewSet(ViewSet):
     permission_classes = (permissions.IsAuthenticated,)
     parser_classes = (MergePatchParser,)
 
+    def get(self, request):
+        """
+        GET /api/user/v1/me
+        """
+        return Response(request.user.username)
```

Comment:
Is this returning a JSON object?
For future compatibility, consider returning the username in a `dict` object.